packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -26,70 +22,66 @@ export default {
       load
     };
   }
-};   
+};
 ```
 
-
 ## Adding item to the cart
 
 To add the product to the cart you can use `addItem` method:
 
 ```vue
 <template>
-  // ...
-    <ul>
-      <li
-        v-for="product in products" :key="product.id"
-      > 
-        // ...
-        <button
-          @click="addItem({ product, quantity })"
-        >
-          Add to cart
-        </button>
-      </li>
-    </ul>
-  // ...
-</template>    
-<script>     
-  import { computed } from '@vue/composition-api';
-  import { useCart } from '{INTEGRATION}';
-  export default {
-    props: {
-      products: {
-        type: Array,
-        required: true
-      }
-    },
-    setup() {
-      const {
-        addItem,
-      } = useCart();
-
-      // load cart if it wasn't loaded before
-
-      return {
-        addItem,
-      };
-    }
-  };
+  <!-- ... -->
+  <ul>
+    <li v-for="product in products" :key="productGetters.getId(product)">
+      <!-- ... -->
+      <input type="number" v-model="quantity" />
+      <button @click="addItem({ product, quantity })">
+        Add to cart
+      </button>
+    </li>
+  </ul>
+  <!-- ... -->
+</template>
+<script>
+import { ref, computed } from '@vue/composition-api';
+import { useCart, useFacet, productGetters, facetGetters } from '{INTEGRATION}';
+
+export default {  
+  setup() {
+    const { addItem } = useCart();
+    const { result } = useFacet();
+
+    // load cart if it wasn't loaded before
+    const quantity = ref(1);
+    const products = computed(() => facetGetters.getProducts(result.value));
+
+    return {
+      addItem,
+      quantity,
+      productGetters,
+      products
+    };
+  }
+};
 </script>
-``` 
+```
 
 ## Removing items and changing their quantity
 
 To remove an item from the cart use `removeItem` method, and similarly to update quantity use `updateItemQty` method:
 
 ```vue
 <template>
+  <!-- ... -->
   <div>
     <ul>
-      <li v-for="product in products" :key="product.id">
-        <input type="number" />
-        <button @click="updateItemQty({ product, quantity })">
+      <li v-for="item in cartItems" :key="cartGetters.getItemSku(product)">
+        <input type="number" v-model="quantity" />
```

Comment:
We loop through `cartItems` but only have a single `quantity`. It's an obvious bug, but implementing it correctly will introduce complexity to this example.

What do you think about changing this code example to call `updateItemQty` on `@blur` (or some other input event), and getting the value from it, instead of relying on `quantity` variable?

Reply:
I wonder if it won't be confusing for users (thinking that they should use @blur here for some reason), so what do you think about adding comment here instead?

Reply:
That's a good point, but I still think it's better than having a code with a bug as an example. What's your take on this @lukaszjedrasik

Reply:
I think we shouldn't show code examples that can create bugs. For me, best solution would be to add `updateItemQty` on `@blur` along with a comment why we are doing this

Reply:
That's okay I will change it for @blur :) thanks for this tip.